web/src/lib/i18n/en.json

```diff
@@ -127,13 +127,14 @@
     "map_enable_description": "Enable map features",
     "map_gps_settings": "Map & GPS Settings",
     "map_gps_settings_description": "Manage Map & GPS (Reverse Geocoding) Settings",
+    "map_implications": "Enabling the map implies clients (web and mobile apps) sending requests to a third-party map server in order to display the map. These requests include the client's IP address, the Immich instance URL, as well as the coordinates and zoom level of the requested map tiles. The default tile provider is tiles.immich.cloud",
```

Comment:
I don't think we need to be so detailed here. The settings page don't need to be 100% self documenting. A general statement is fine and more details can live at immich.app. I would prefer to see something like:

> The map feature relies on external service to function properly. Read more about those implications <link>here</link>